test/data/manifests/fedora_36-aarch64-iot_raw_image-boot.json

```diff
@@ -2389,6 +2396,130 @@
                 {
                   "from": "input://root-tree/",
                   "to": "mount://root/"
+                },
+                {
+                  "from": "input://ostree-tree///usr/lib/ostree-boot/efi/bcm2710-rpi-2-b.dtb",
```

Comment:
The triple slash `/` does not look correct to be honest 🤔

Reply:
The triple slash in the test manifests is because 'normally' the commit ID is in between there making the full path `input://ostree-tree/1337abcd1337//usr/lib/ostree-boot/efi/bcm2710-rpi-2-b.dtb` which still has a *double* slash but that isn't the worst?

Reply:
Thanks for the additional info. The double slash still looks weird and I would be interested to know where it is coming from. But If this does not cause issues when building images, I could accept it. It would be however good to not have it there if possible.